test/shared/src/main/scala-2.x/zio/test/AssertionVariants.scala

```diff
@@ -20,14 +20,15 @@ import zio.test.Assertion.Render._
 
 trait AssertionVariants {
 
+  final def equals[A, B](b: B, a: A)(implicit eql: Eql[A, B]): Boolean =
```

Comment:
Does this actually give you any type safety? I think the compiler is just going to widen the types here.

Reply:
I believe the main reason for defining `equals` was so that `equalTo` and `equalToOneOf` had the same behaviour when testing equality of arrays.

Reply:
If you just want to do that then I would suggest defining a helper method that takes a single type parameter. Using the multiple type parameters and the `Eql` is trying to do some more type level machinery for the `equalTo` assertion in particular so if we just want to handle arrays we can simplify substantially.